packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -241,7 +241,7 @@ export function createTestEsCluster<
         ).installPath;
       } else if (esFrom === 'snapshot') {
         installPath = (await firstNode.installSnapshot(config)).installPath;
-      } else if (esFrom === 'serverless') {
+      } else if (esFrom === 'serverless' && esServerlessOptions) {
```

Comment:
I think without `esServerlessOptions` we are not able to start Elasticsearch properly, but curious if this check should be done explicitly before `firstNode.runServerless` call. WDYT?

Reply:
:+1: if there's a dev facing reason this may surface, with an explicit check we can add a log message